src/Microsoft.DocAsCode.Build.Engine/DocumentBuildContext.cs

```diff
@@ -43,14 +42,16 @@ public DocumentBuildContext(string buildOutputFolder, IEnumerable<FileAndType> a
             string versionName,
             ApplyTemplateSettings applyTemplateSetting,
             string rootTocPath,
-            string versionFolder)
+            string versionFolder,
+            ImmutableArray<string> xrefserviceUrls)
```

Comment:
yes, i made a mistake

---
In reply to: [127389377](https://github.com/dotnet/docfx/pull/1837#discussion_r127389377) [](ancestors = 127389377)